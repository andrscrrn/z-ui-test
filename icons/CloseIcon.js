import React from "react";

export default function CloseIcon() {
  return (
    <svg width="15px" height="16px">
      <path
        fillRule="evenodd"
        fill="rgb(51, 51, 51)"
        d="M15.010,14.487 L14.318,15.176 L7.828,8.706 L0.696,15.815 L-0.010,15.111 L7.122,8.003 L0.632,1.534 L1.323,0.845 L7.813,7.314 L13.968,1.179 L14.674,1.883 L8.519,8.017 L15.010,14.487 Z"
      />
    </svg>
  );
}
