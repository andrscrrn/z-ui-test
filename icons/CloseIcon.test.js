import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

import CloseIcon from "./CloseIcon";

describe("CloseIcon", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<CloseIcon />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
