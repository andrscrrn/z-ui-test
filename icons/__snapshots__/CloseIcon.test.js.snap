// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CloseIcon renders correctly 1`] = `
<svg
  height="16px"
  width="15px"
>
  <path
    d="M15.010,14.487 L14.318,15.176 L7.828,8.706 L0.696,15.815 L-0.010,15.111 L7.122,8.003 L0.632,1.534 L1.323,0.845 L7.813,7.314 L13.968,1.179 L14.674,1.883 L8.519,8.017 L15.010,14.487 Z"
    fill="rgb(51, 51, 51)"
    fillRule="evenodd"
  />
</svg>
`;
